coreTable/CoreTable.ts

```diff
@@ -70,6 +70,7 @@ const TransformsRequiringCompute = new Set([
     TransformType.AppendRows,
     TransformType.AppendColumns,
     TransformType.UpdateRows,
+    TransformType.UpdateColumnDefs,
```

Comment:
@breck7 you made `toPercentageFromEachEntityForEachTime` column-based in https://github.com/owid/owid-grapher/commit/f591fe106f37efddf1834148ed1121d1a5661e89#diff-d0c27594f617591ec24495871c2d1eb77b60e4debc13c45614cae1fb16c887f7R303-R306, but didn't add `UpdateColumnDefs` to `TransformsRequiringCompute`. I hope this is the right thing to do?

Reply:
Yeah we were discussing this: https://owid.slack.com/archives/C01D295TM4K/p1603838545030400?thread_ts=1603830711.024300&cid=C01D295TM4K

But not sure what the right approach is!

Reply:
Ugh, I really don't like `TransformsRequiringCompute` and think we should probably get rid of that ugly mess. I think what maybe will work better is this:

1. Get rid of `fn` on column definition. Instead transforms should just pass in a new transformed columnStore, or a `values` array on column defs, or a `transform` string which is basically a run-time DSL for describing transforms.
2. Then all 3 of those patterns would be column based and fast, and we can ditch TransformsRequiringCompute and just always build the columnStore in the constructor immediately (well, maybe we could lazily compute `transforms` if that's every a perf issue later on) .

Reply:
So for now, if the tests pass and it doesn't throw perf off noticeably, this is fine. I think the only thing that may be worse perf wise here would be CovidExplorer v1, but hopefully we can nuke that code today or tomorrow.